pkgs/development/python-modules/zulip/default.nix

```diff
@@ -26,12 +29,22 @@ buildPythonPackage rec {
     requests
     matrix-client
     distro
+
+    # from requests[security]
+    cryptography
+    pyopenssl
```

Comment:
Normally such dependencies belong into requests but it is a more widely used package so it could create dependency loops and blow up the size of the module. @jonringer what do you think is the best solution here?

Reply:
The root cause is 2fd38834a82a0b78681d1b9f9005d85ec0b243f1.

Reply:
Master still has pyopenssl. Just certify was removed https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/urllib3/default.nix#L10

Reply:
If anything we could add those packages to requests' `propagatedBuildInputs`.
But I don't see what's wrong with doing it for every package. Requests is used a lot so it's advantageous if it has a smaller closure size.

Reply:
Yeah. Not sure how to properly handle this. Maybe we can add a requests-security if many packages require those extra packages.

Reply:
If you notice anything else breaking, let me know.